scrapy/utils/log.py

```diff
@@ -122,7 +122,14 @@ def _get_handler(settings):
     filename = settings.get('LOG_FILE')
     if filename:
         encoding = settings.get('LOG_ENCODING')
-        handler = logging.FileHandler(filename, encoding=encoding)
+        if settings.get("LOG_ROTATING") is True:
```

Comment:
```suggestion
        if settings.getbool("LOG_ROTATING") is True:
```

This makes it possible to set the setting from the command line, where every value is a string.